src/types/Canvas.d.ts

```diff
@@ -0,0 +1,26 @@
+import Konva from 'konva';
+import { Tool } from '../tools/Tool'
+
+export type VueKonvaLayer = Konva.Layer & Vue & {
+  getNode: Function
+}
+
+export type VueKonvaStage = Konva.Stage & Vue & {
+  getNode: Function
+}
+
+export interface Data {
+  x: number;
+  y: number;
+}
+
+export interface CanvasElementPayload {
+  tool: Tool;
+  immediate: boolean;
+  data: Data[];
+}
+
+export interface CanvasElement extends CanvasElementPayload{
+  id: string;
+  jti: string;
```

Comment:
What's a jti? Sould include some comments to clarify

Reply:
it is the unique user id

Reply:
@Ataww @bryansamuel1 https://tools.ietf.org/html/rfc7519#section-4.1.7